PROJECT_FUND/2023.md

```diff
@@ -0,0 +1,4 @@
+# 2023 Project Fund Request Record
+
+First Name | Last Name | Project | Amount requested | Pull Request date | Pull Request link | Date Expense Report sent | Amount of Expense Report
+-|-|-|-|-|-|-|-
```

Comment:
```suggestion
First Name | Last Name | Project | Amount requested | Pull Request date | Pull Request link
-|-|-|-|-|-
```

The original issue #953 suggested that when approved, funds would be transferred to a crowdfunding platform from which the project itself may manage their further distribution. This practice is also implied by the "Distribution of the Funds" section of the PR template.

If that is still the plan, the expense report references here are out of place.

Reply:
Gotcha, I have forgotten about that. Should we add a field for linking to the crowdfunding platform? (Like basically a link for the campaign?) I bet it would be something like OpenCollective?